Gordon360/Services/ProfileService.cs

```diff
@@ -3,22 +3,31 @@
 using Gordon360.Models.CCT;
 using Gordon360.Models.ViewModels;
 using Newtonsoft.Json.Linq;
+using Microsoft.Extensions.Configuration;
 using System;
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading.Tasks;
+using System.Net.Mail;
+using System.Net;
 
 namespace Gordon360.Services
 {
     public class ProfileService : IProfileService
     {
         private readonly CCTContext _context;
         private readonly IAccountService _accountService;
+        private readonly IConfiguration _config;
+        private static EmailAccountViewModel serviceEmail;
 
-        public ProfileService(CCTContext context)
-        {
+        public ProfileService(CCTContext context, IConfiguration config, IAccountService accountService)
+        { 
             _context = context;
-            _accountService = new AccountService(context);
+            _config = config;
+            serviceEmail = new EmailAccountViewModel();
+            serviceEmail.Username = _config["Emails:Default:Username"];
+            serviceEmail.Password = _config["Emails:Default:Password"];
```

Comment:
```suggestion
            serviceEmail = new EmailAccountViewModel() {
            Username = _config["Emails:Default:Username"],
            Password = _config["Emails:Default:Password"]
            }
```
I can't indent it correctly...but this is another clean option for initializing these values. Feel free to stick with what you have! Just want to make sure you know the options.

Reply:
ahhh I see, I somehow completely forgot that it was a constructor is a constructor. I've been working with a few too many default constructors. I'll indent it properly and update this.